src/mealList.js

```diff
@@ -0,0 +1,317 @@
+  for (let i = 0; i <= data.length - 1; i += 1) {
+    const foodDiv = document.createElement('div');
+    foodDiv.classList.add('col-lg-4');
+
+    const listItem = document.createElement('div');
+    listItem.id = data[i].idMeal;
+    foodDiv.setAttribute('data-id', `${data[i].idMeal}`);
+    listItem.classList.add('meal-item', 'text-center', 'card', 'shadow', 'mb-4', 'border-0');
+
+    const itemImage = document.createElement('div');
+    itemImage.classList.add('meal-img');
+
+    const image = document.createElement('img');
+    image.src = data[i].strMealThumb;
+
+    itemImage.appendChild(image);
+    listItem.appendChild(itemImage);
+
+    const itemText = document.createElement('div');
+    itemText.classList.add('meal-name', 'd-flex', 'pt-3', 'justify-content-center', 'align-items-center');
+
+    const mealName = document.createElement('h3');
+    mealName.classList.add('fs-6', 'me-2', 'pt-1');
+    mealName.innerHTML = data[i].strMeal;
+
+    const likeBtn = document.createElement('i');
+    likeBtn.classList.add('bi', 'bi-heart', 'like-btn', 'text-danger');
+    likeBtn.id = data[i].idMeal;
+
+    itemText.appendChild(mealName);
+    itemText.appendChild(likeBtn);
+    listItem.appendChild(itemText);
+
+    const mealLikes = document.createElement('div');
+    const numOfLikes = document.createElement('small');
+    numOfLikes.innerHTML = '0 likes';
+    mealLikes.appendChild(numOfLikes);
+    listItem.appendChild(mealLikes);
+
+    const commentContainer = document.createElement('div');
+    commentContainer.classList.add('comment', 'mb-4', 'mt-2');
+
+    const cmtButton = document.createElement('button');
+    cmtButton.classList.add('btn', 'btn-secondary', 'rounded-pill', 'px-4', 'shadow', 'comment-btn');
+    cmtButton.setAttribute('id', `${data[i].idMeal}`);
+    cmtButton.setAttribute('type', 'button');
+    cmtButton.setAttribute('data-bs-toggle', 'modal');
+    cmtButton.setAttribute('data-bs-target', '#staticBackdrop');
+    cmtButton.innerHTML = 'comments';
+
+    commentContainer.appendChild(cmtButton);
+    listItem.appendChild(commentContainer);
+
+    const resevationContainer = document.createElement('div');
+    resevationContainer.classList.add('comment', 'mb-4', 'mt-2');
+
+    const rsvButton = document.createElement('button');
+    rsvButton.classList.add('btn', 'btn-secondary', 'rounded-pill', 'px-4', 'shadow', 'reservation-btn');
+    rsvButton.setAttribute('id', `${data[i].idMeal}`);
+    rsvButton.setAttribute('type', 'button');
+    rsvButton.setAttribute('data-bs-toggle', 'modal');
+    rsvButton.setAttribute('data-bs-target', '#staticBackdrop');
+    rsvButton.innerHTML = 'reservation';
+
+    resevationContainer.appendChild(rsvButton);
+    listItem.appendChild(resevationContainer);
+
+    foodDiv.appendChild(listItem);
+    mainDiv.appendChild(foodDiv);
+
+    foodDiv.appendChild(listItem);
+    mainDiv.appendChild(foodDiv);
+
+    // eslint-disable-next-line no-await-in-loop
```

Comment:
- [ ] please like the previous reviewer noted, eslint is necessary with the goal of making code more consistent and avoiding bugs. so disabling it only means there is an issue. remove and solve the error like he said. great job